Neos.ContentGraph.PostgreSQLAdapter/src/Domain/Projection/Feature/NodeRenaming.php

```diff
@@ -31,10 +32,12 @@ trait NodeRenaming
     public function whenNodeAggregateNameWasChanged(NodeAggregateNameWasChanged $event): void
     {
         $this->transactional(function () use ($event) {
-            foreach ($this->getProjectionHyperGraph()->findNodeRecordsForNodeAggregate(
-                $event->getContentStreamIdentifier(),
-                $event->getNodeAggregateIdentifier()
-            ) as $originNode) {
+            foreach (
+                $this->getProjectionHyperGraph()->findNodeRecordsForNodeAggregate(
+                    $event->getContentStreamIdentifier(),
```

Comment:
i guess these `NodeAggregateTypeWasChanged` getters can be switched to property access too - ... sometime

Reply:
yep, will be done when NodeMove is touched next time